cavern/VERSION

```diff
@@ -1,4 +1,4 @@
 ## deployable containers have a semantic and build tag
 # semantic version tag: major.minor
 # build version tag: timestamp
-TAGS="0.2.6 $(date -u +"%Y%m%dT%H%M%S")"
+TAGS="0.2.7 $(date -u +"%Y%m%dT%H%M%S")"
```

Comment:
We discovered that version tags are useful if there is onen tag with the semantic version and the timestamp, so the above should have a dash, eg `0.2.7-$(date ...)`

That makes it much easier to track a running container back to the image and have one tagged image per build.

Reply:
That would make testing in keel-dev a bit of a pain.  The "imagePullPolicy" in k8s that I've found most useful and least dangerous is "Always".  It's the only one that compares the image checksum in the remote registry.  Adding the timestamp to the version would mean the image URI would have to be updated on each change.  Also, the image SHA is there to precisely identify the build.

Reply:
I'm open to changing this but this is the way it's done in keel at the moment.  There will be a point I suppose when ops takes over the configuration and they will have to reconcile the different approaches.

Reply:
OK, it's really up to operational constraints. But you must not be using either of the tags then, because I only proposed tagging once with both pieces of info instead of twice. How do you revert to a previous image when there is a broken image?

Reply:
Testing/iteration is done on keel-dev using the same tagged version eg :1.3.  When ready to release, the keel-prod config is changed from 1:2 to 1.3 and can be reverted if needed.  I probably haven't explained this very well!